settings.py

```diff
@@ -67,6 +67,7 @@ def insert_debug_toolbar_middleware(middlewares):
 
 SITE_URL = os.environ.get('OLYMPIA_SITE_URL') or 'http://localhost:8000'
 DOMAIN = SERVICES_DOMAIN = urlparse(SITE_URL).netloc
+ADDONS_FRONTEND_PROXY_PORT = '7000'
```

Comment:
Since `settings_test.py` loads settings from `settings.py`, we're going to end up with ``ADDONS_FRONTEND_PROXY_PORT` being defined in tests. Can you set it to `None` in `settings_tests.py` as well ?